doc/changelog/09-coqide/12874-show_proof_diffs.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  Support for "Show Proof Diffs" in CoqIDE from the View menu
```

Comment:
This should be updated to reflect the no diff by default. Maybe something on this basis:
```suggestion
  Support for :cmd:"Show Proof" in CoqIDE from the :n:`View` menu, showing differences from the previous state if the :n:`Diffs` mode is set
```
(I don't know exactly what modifiers to use in the rst syntax though.)

Reply:
Rst takes a backtick here and in many other cases, so it would be ``` :cmd:`Show Proof` ```

The doc already explains your suggested addition.  The change log doesn't need to explain every detail or be utterly precise--it's just a teaser.

```
Support showing diffs for :cmd:`Show Proof` in CoqIDE from the :n:`View` menu.
See :ref:`showing_proof_diffs`.
```

Reply:
Yes, that's better.